README.md

```diff
@@ -129,7 +141,19 @@ The data is returned as a list of objects (seen below).
     isVerified: Boolean,
     isOfficialArtist: Boolean,
     hasOwnerReplied: false,
-    replyToken: null
+    replyToken: null,
+    customEmojis: Array [ // An Array of custom emojis used in the comment
+      {
+        text: String, // the text alias for the emoji
+        emojiThumb: Array [ // An Array of thumbnails of the custom emoji
```

Comment:
It's a collection, so including `s` at the end is important for me.
Just like `customEmojis`
(Other places need renaming too)
```suggestion
        emojiThumbnails: Array [ // An Array of thumbnails of the custom emoji
```

Reply:
I chose "emojiThumb" because it is formatted the same as "authorThumb". I don't think authorThumb should be changed because other projects may be using it.

Reply:
I am not suggesting `authorThumb` or any other **existing** fields to be renamed.
I am suggesting `emojiThumb` needs to be renamed not just in this line but also other places/files :P

Reply:
Done